docs/modules/servers/pages/distributed/operate/metrics.adoc

```diff
@@ -4,12 +4,22 @@
 James relies on the https://metrics.dropwizard.io/4.1.2/manual/core.html[Dropwizard metric library]
 for keeping track of some core metrics of James.
 
-Such metrics are made available via JMX. You can connect for instance using VisualVM and the associated
-mbean plugins.
+Such metrics are exposed over HTTP and can be made available by using **extensions.routes** in https://github.com/apache/james-project/blob/master/docs/modules/servers/pages/distributed/configure/webadmin.adoc[webadmin.properties] file:
+....
+extensions.routes=org.apache.james.webadmin.dropwizard.MetricsRoutes
+....
+
+Once exposed, you can use https://prometheus.io/[Prometheus] to scrape and visualize them with https://grafana.com/[Grafana].
+Below here is the sample scrape job in **prometheus.yml** file:
+....
+  - job_name: 'Apache James'
+    scrape_interval: 30s
+    metrics_path: /metrics
+    static_configs:
+      - targets: ['james:8000']
+....
 
-You can also export the metrics to ElasticSearch and visualize them with https://grafana.com/[Grafana].
-See xref:distributed/configure/elasticsearch.adoc#_exporting_metrics_directly_to_elasticsearch[elaticseach.properties]
-documentation for more details on how to set this up.
+You can also export the metrics to ElasticSearch and visualize them in https://grafana.com/[Grafana] dashboards.
```

Comment:
Why changing this part?

Reply:
Ok I can put the ElasticSearch/Grafana section back